src/main/java/io/netty/incubator/codec/quic/QuicChannel.java

```diff
@@ -165,6 +165,13 @@ default ChannelPromise voidPromise() {
      */
     long peerAllowedStreams(QuicStreamType type);
 
+    /**
+     * Returns {@code true} if the connection was closed because of the idle timeout.
+     *
+     * @return {@code true} if the connection was closed because of the idle timeout, {@code false}.
```

Comment:
```suggestion
     * Returns {@code true} if the connection was closed because of idle timeout.
     *
     * @return {@code true} if the connection was closed because of idle timeout, {@code false}.
```